scss/components/_flex.scss

```diff
@@ -32,3 +32,30 @@
     }
   }
 }
+
+@mixin gimbal($o:row,$x:left,$y:top) {
```

Comment:
"Orientation" of natural directions remains constant regardless of whether or not the flex container flexes row or flexes column :)
[real world example](https://www.youtube.com/watch?v=29qeHVT-8-I)